input/templates/partials/catalog/quickview.hbs

```diff
@@ -43,7 +44,7 @@
             </div>
             <div class="quickview-wishlist-row">
               <ul class="list-inline">
-                <li>
+                <li {{#if wishlist}}class="hidden"{{/if}}>
```

Comment:
This condition hides the "add to wishlist" button when the quickview is viewed in the wishlist. Meaning that I intentionally added the `#if`here.